vissl/data/data_helper.py

```diff
@@ -24,6 +24,65 @@ def get_mean_image(crop_size):
     return img
 
 
+@contextlib.contextmanager
+def with_temporary_numpy_seed(seed: int):
```

Comment:
nit: rename `sampling_seed` for clarity that it's not the other training seed :)